airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml

```diff
@@ -0,0 +1,1609 @@
+                          properties:
+                            accepted:
+                              type: number
+                            delivered:
+                              type: number
+                            finished:
+                              type: number
+                            kind:
+                              type: string
+                            planned:
+                              type: number
+                            rejected:
+                              type: number
+                            started:
+                              type: number
+                            unscheduled:
+                              type: number
+                            unstarted:
+                              type: number
+                          type: object
+                      type: object
+                    current_state:
+                      type: string
+                    follower_ids:
+                      type: array
+                    owner_ids:
+                      type: array
+                    status:
+                      type: string
+                    updated_at:
+                      type: string
+                  type: object
+                story_priority:
+                  type: string
+                story_type:
+                  type: string
+              type: object
+            type: array
+          guid:
+            type: string
+          highlight:
+            type: string
+          kind:
+            type: string
+          message:
+            type: string
+          occurred_at:
+            type: string
+          performed_by:
+            properties:
+              id:
+                type: number
+              initials:
+                type: string
+              kind:
+                type: string
+              name:
+                type: string
+            type: object
+          primary_resources:
+            items:
+              properties:
+                id:
+                  type: number
+                kind:
+                  type: string
+                name:
+                  type: string
+                story_priority:
+                  type: string
+                story_type:
+                  type: string
+                url:
+                  type: string
+              type: object
+            type: array
+          project:
+            properties:
+              id:
+                type: number
+              kind:
+                type: string
+              name:
+                type: string
+            type: object
+          project_version:
+            type: number
+          secondary_resources:
+            type: array
+        type: object
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://www.pivotaltracker.com/services/v5/
+        path: projects/{{ stream_slice['project_id'] }}/activity?limit=10
```

Comment:
The `path` parameter should not contain query params. Please handle this via pagination. In addition, this stream has less records than the original one.

Reply:
It is a little workaround, as cursor Pagination field from response gives full next page path, it is directly injected to path
The newly injected path should not contain any limit parameter.

Reply:
The Pagination is properly handled via Cursor Pagination block